### DISCUSSION TITLE
RaspAP with two external antennas

### DISCUSSION THREAD

**Discussion by christian1211** [RaspAP/raspap-webgui#1020] [Q&A]
Hello all,

I set up RaspAP on a fresh raspberry. I connected two external antennas and managed to use one for connecting as a client with a wifi and the second one for setting up a hotspot. When I check the UI, everything is fine but the pi itself has no internet and also I cannot connect to the hotspot from the pi. I don't know where is the error. I can provide logs if I know which ones are needed. Please help, I got no more ideas left.

**Comment by billz** [comment]
See this discussion https://github.com/RaspAP/raspap-webgui/discussions/1022
